packages/input-container/src/vaadin-input-container.js

```diff
@@ -19,6 +19,17 @@ export class InputContainer extends ThemableMixin(DirMixin(PolymerElement)) {
           display: flex;
           align-items: center;
           flex: 0 1 auto;
+          /* For fields that flip */
+          border-start-start-radius: var(--vaadin-input-container-top-start-radius, var(--_border-radius));
+          border-start-end-radius: var(--vaadin-input-container-top-end-radius, var(--_border-radius));
+          border-end-start-radius: var(--vaadin-input-container-bottom-start-radius, var(--_border-radius));
+          border-end-end-radius: var(--vaadin-input-container-bottom-end-radius, var(--_border-radius));
+          /* For non-flipped fields */
+          border-top-left-radius: var(--vaadin-input-container-top-left-radius, var(--_border-radius));
```

Comment:
Setting a value for example to `--vaadin-input-container-top-left-radius` doesn't work because the default value of `--vaadin-input-container-top-start-radius` will override it

Reply:
In case of `vaadin-date-time-picker` it worked, see https://github.com/vaadin/web-components/pull/5251/commits/25977b3502ae9c31f554fd61fbd5d8a345f5bf8d. Or am I missing something?
If both are set, there will be a problem. I'm not sure if this is an acceptable tradeoff.

Reply:
Ah, sorry, it's the other way around in the latest revision. So setting a value to `--vaadin-input-container-top-start-radius` doesn't do anything because the default value of `--vaadin-input-container-top-left-radius` overrides it:

![Screenshot 2023-01-11 at 11 31 03](https://user-images.githubusercontent.com/1222264/211770137-01fe32e6-a108-4057-84c9-aa86a8a17dc8.png)

![Screenshot 2023-01-11 at 11 31 17](https://user-images.githubusercontent.com/1222264/211770152-f4c7e5cf-8ae7-4056-921a-c525cba1166c.png)

Reply:
Then it's definitely a problem that I don't know how to solve, except if we stick to `-left` and `-right`.
@jouni what do you think about this? Should we avoid adding CSS logical props for now?

Reply:
Apart from the issue where and how we should define the component-specific custom properties, the solution I would go with is that we only expose the logical properties in the public API, and map them internally to the directional properties to support browsers that don't support logical props.

For example:

```css
:host {
  border-start-start-radius: var(--vaadin-input-container-top-start-radius);
  border-start-end-radius: var(--vaadin-input-container-top-end-radius);
  ...
}

@supports not (border-start-start-radius: 1px) {
  :host {
    border-top-left-radius: var(--vaadin-input-container-top-start-radius);
    border-top-right-radius: var(--vaadin-input-container-top-end-radius);
    ...
  }

  :host([dir="rtl"]) {
    border-top-left-radius: var(--vaadin-input-container-top-end-radius);
    border-top-right-radius: var(--vaadin-input-container-top-start-radius);
    ...
  }
}

```

Reply:
@jouni Thank you, updated the PR accordingly.

With the current approach of not enabling globals by default, should we actually copy these to Lumo? 🤔